source/includes/connect.cpp

```diff
@@ -0,0 +1,15 @@
+// start-instance
+#include <mongocxx/instance.hpp>
+
+mongocxx::instance instance{};
+// end-instance
+
+// start-client
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+#include <mongocxx/client.hpp>
+
+mongocxx::instance instance{};
```

Comment:
Q: Should we have a link to the instance page somewhere around this? I worry that since it's included in the code example, someone might add one again thinking it's needed for for the connection. Which it kind of is, but also isn't if you already have one instantiated earlier (if I understand right)?

Reply:
yeah, i realized we probably shouldn't include it in all the code examples, but maybe use an admonition to let people know that they need it